.pre-commit-config.yaml

```diff
@@ -64,6 +64,25 @@ repos:
     hooks:
     -   id: pylint
         stages: [manual]
+-   repo: https://github.com/pycqa/pylint
+    rev: v2.15.5
+    hooks:
+    -   id: pylint
+        alias: redefined-outer-name
+        name: Redefining name from outer scope
```

Comment:
What's the idea of adding this one that only runs a specific check if we already have a pylint one for all checks above (since both are labeled as "manual" stage)

Reply:
this one can't be applied to all files, it has lots of false positives in tests because of fixtures

but, like this, we can turn it on for the non-test files

unfortunately pylint doesn't have a per-file-excludes option like flake8 does

Reply:
Ah, OK, thanks for the explanation!